feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/presentation/importing/source/source/RawSeedImportSource.kt

```diff
@@ -34,17 +37,30 @@ class RawSeedImportSource(
 
     override val fieldsValidFlow: Flow<Boolean> = rawSeedFlow.map { it.isNotEmpty() }
 
+    init {
+        scanSeedRequester.responseFlow
+            .onEach {
+                // A short delay to wait until view will be attached to fragment. Otherwise the input view clear provided value on attach to fragment
+                delay(150)
```

Comment:
We should probably just remove saving instnace state for input field. We did the same for SwapSettingsFragment when dealing with input fields after selecting a swap asset